README.md

```diff
@@ -30,3 +34,37 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+//Worked Directly With Aaron Krapes in person
+
+$$ T(n) =
+   \begin{cases}
+       1 & n \leq 1\\
+       3T(\frac{n}{3}) + n^5 & n > 1
+   \end{cases}
+$$
+
+$T(n) = 3T(\frac{n}{3}) + n^5 $
+
+$= 3(3T(\frac{n}{9}) + \frac{n^5}{3^5}) + n^5$
+
+$= 9T(\frac{n}{9}) + \frac{n^5}{3^4} + n^5$
+
+$= 27T(\frac{n}{27}) + \frac{n^5}{9^4} + \frac{n^5}{3^4} + n^5$
+
+$= 27T(\frac{n}{27}) + \frac{n^5}{3^8} + \frac{n^5}{3^4} + n^5$
+
+$…$
+
+$= 3^iT(\frac{n}{3^i}) +$
+$(\sum_{k=0}^i (\frac{1}{3^4})^k)$
+$\cdot n^5 $
+
+Note: $\sum_{k=0}^i (\frac{1}{3^4})^k = 1 +(\frac{1}{3^4})^1 +(\frac{1}{3^4})^2 ... (\frac{1}{3^4})^i$ 
+
+for $i = \log n$
+
+so $(\sum_{k=0}^i (\frac{1}{3^4})^k) \cdot n^5 = (1 +(\frac{1}{3^4})^1 +(\frac{1}{3^4})^2 + ... +(\frac{1}{3^4})^{\log n}) \cdot n^5 = (1 +(\frac{1}{3^4})^1 +(\frac{1}{3^4})^2 + ... +(\frac{1}{3^4})n) \cdot n^5 = n^5 +(\frac{1}{3^4})^1 \cdot n^5 +(\frac{1}{3^4})^2 \cdot n^5 + ... +(\frac{1}{3^4})n \cdot n^5 = n^5 +(\frac{1}{3^4})^1 \cdot n^5 +(\frac{1}{3^4})^2 \cdot n^5 + ... +(\frac{1}{3^4})n^6 $
```

Comment:
Where does the $n^6$ come from? As far as I can see you don't have any terms with n that are multiplied by $n^5$ in the pattern above.

Reply:
Near the end, $n^5 +(\frac{1}{3^4})^1 \cdot n^5 +(\frac{1}{3^4})^2 \cdot n^5 + ... +(\frac{1}{3^4})n \cdot n^5$ would $\frac{1}{3^4})n \cdot n^5$ not equal $(\frac{1}{3^4})n^6$ ?